tests/cases/fourslash/incompleteFunctionCallCodefixTypeKeyof.ts

```diff
@@ -15,7 +15,7 @@ verify.codeFix({
   added(keyofTypeof);
 }
 
-function added(keyofTypeof: string | number | symbol) {
+function added<T>(keyofTypeof: keyof T) {

```

Comment:
This isn't a particularly great codefix... since `keyof T` when `T` doesn't appear "naked" somewhere in the parameters list is usually a lousy parameter. I don't think it's worth fighting it here though as it *could* appear naked somewhere.